api/client.go

```diff
@@ -14,6 +14,10 @@ import (
 
 	genq "github.com/Khan/genqlient/graphql"
 	"github.com/superfly/graphql"
+	"go.opentelemetry.io/otel"
```

Comment:
There's a lint error on this:

```
import of package `go.opentelemetry.io/otel` is blocked because the module is not in the allowed modules list. (gomodguard)
```

Any idea what that's about?

Reply:
This is `gomodguard` linter enforcing the dependency tree is kept to a minimal because then it is imported from flaps/flyd and things go bad. That said, adding `otel` is a safe bet here.

https://github.com/superfly/flyctl/blob/4a391eb3b88d1432520670b848154c4f41513fc7/api/.golangci.yml#L46-L51